manifests/repo.pp

```diff
@@ -4,8 +4,8 @@
 #
 #   include homebrew::repo
 class homebrew::repo (
-  $installdir   = $homebrew::config::installdir,
-  $min_revision = $homebrew::config::min_revision,
+  $installdir   = $homebrew::installdir,
+  $min_revision = $homebrew::min_revision,
```

Comment:
That seems to be intentional; it looks like you're undoing that? Can you explain how this relates to this PR?